backend/btrixcloud/db.py

```diff
@@ -96,7 +95,7 @@ async def update_and_prepare_db(
     await ping_db(mdb)
     print("Database setup started", flush=True)
     if await run_db_migrations(
-        mdb, user_manager, page_ops, org_ops, background_job_ops, coll_ops
```

Comment:
Quite a few of the migrations use `page_ops` so we can't drop it here, we should only drop it from `craete_indexes` below.